subsys/net/ip/net_private.h

```diff
@@ -24,7 +24,7 @@
  * NOTE: Update comments here and calculate which struct occupies max size.
  */
 
-#ifdef CONFIG_NET_L2_WIFI_MGMT
+#if defined(CONFIG_NET_L2_WIFI_MGMT) || defined (CONFIG_NET_L2_NRF_WIFI_MGMT)
```

Comment:
```suggestion
#if defined(CONFIG_NET_L2_WIFI_MGMT) || defined(CONFIG_NET_L2_NRF_WIFI_MGMT)
```

Reply:
I don't think this is resolved yet. The diff from push is empty.

Reply:
oops, sorry, missed `-a` in `--amend`